ansible/MANUAL_STEPS.md

```diff
@@ -229,7 +229,9 @@ mount /aha
 ```bash
 cd /opt/ibm-xlc
 uncompress 16.1.0.3-IBM-xlCcmp-AIX-FP003.tar.Z
+tar -xvf 16.1.0.3-IBM-xlCcmp-AIX-FP003.tar
 uncompress IBM_XL_C_CPP_V16.1.0.0_AIX.tar.Z
```

Comment:
Just curious, should/does the filename here match the version (`1.6.0.3` from the earlier package)?

Reply:
No in typical IBM fashion you download the base package first (16.1.0) and then apply the fix packs on top (16.1.0.3)